server/app.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const app = express();
+const PORT = 4000;
+const router = require("./routes");
+
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+app.use("/", router);
```

Comment:
```suggestion
app.use(router);
```
langsung juga bisa yah